.kokoro

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Copyright 2017-present The Material Components for iOS Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Fail on any error.
+set -e
+
+# Display commands to stderr.
+set -x
+
+KOKORO_RUNNER_VERSION="v3.*"
+
+fix_bazel_imports() {
+  if [ -z "$KOKORO_BUILD_NUMBER" ]; then
+    tests_dir_prefix=""
+  else
+    tests_dir_prefix="github/repo/"
+  fi
+  
+  rewrite_source() {
```

Comment:
does running kokoro result in file system changes?

Reply:
While this script is running it needs to perform some amount of source modification in order for swift import statements to point to the bazel naming convention of `path_to_target_target`. The script makes the modifications before running bazel and then reverts the modifications once the script terminates.